walkthrough/snapshot.py

```diff
@@ -90,6 +90,8 @@ def app(run_checks: bool) -> None:
 
     po.put_markdown("# Walkthrough - Snapshot")
     with open(CURRENT_DIR / "snapshot.md", "r") as f:
+        if not utils.WALKTHROUGH_ORIGINS:
+            po.put_info(po.put_markdown("To use **Origins**, call it with `WALKTHROUGH_ORIGINS=1 walkthrough ...` "))
```

Comment:
Should the logic now be reversed? I think now `WALKTHROUGH_ORIGINS` should be 1 by default.

Reply:
You're right, all in!